releases/1.9/edge/bundle.yaml

```diff
@@ -0,0 +1,313 @@
+    _github_repo_branch: main
+  kfp-db:
+    charm: mysql-k8s
+    channel: 8.0/stable
+    scale: 1
+    trust: true
+    constraints: mem=2G
+    _github_dependency_repo_name: mysql-k8s-operator
+    _github_dependency_repo_branch: main
+  kfp-metadata-writer:
+    charm: kfp-metadata-writer
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-persistence:
+    charm: kfp-persistence
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-profile-controller:
+    charm: kfp-profile-controller
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-schedwf:
+    charm: kfp-schedwf
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-ui:
+    charm: kfp-ui
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-viewer:
+    charm: kfp-viewer
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  kfp-viz:
+    charm: kfp-viz
+    channel: 2.2/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+    _github_repo_branch: main
+  knative-eventing:
+    charm: knative-eventing
+    channel: 1.12/edge
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-eventing
+    _github_repo_name: knative-operators
+    _github_repo_branch: main
+  knative-operator:
+    charm: knative-operator
+    channel: 1.12/edge
+    scale: 1
+    trust: true
+    _github_repo_name: knative-operators
+    _github_repo_branch: main
+  knative-serving:
+    charm: knative-serving
+    channel: 1.12/edge
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-serving
+      istio.gateway.namespace: kubeflow
+      istio.gateway.name: kubeflow-gateway
+    _github_repo_name: knative-operators
+    _github_repo_branch: main
+  kserve-controller:
+    charm: kserve-controller
+    channel: 0.13/edge
+    scale: 1
+    trust: true
+    options:
+      deployment-mode: serverless
+    _github_repo_name: kserve-operators
+    _github_repo_branch: main
+  kubeflow-dashboard:
+    charm: kubeflow-dashboard
+    channel: 1.9/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-dashboard-operator
+    _github_repo_branch: main
+  kubeflow-profiles:
+    charm: kubeflow-profiles
+    channel: 1.9/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-profiles-operator
+    _github_repo_branch: main
+  kubeflow-roles:
+    charm: kubeflow-roles
+    channel: 1.9/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-roles-operator
+    _github_repo_branch: main
+  kubeflow-volumes:
+    charm: kubeflow-volumes
+    channel: 1.9/edge
```

Comment:
Right now this track doesn't exist

Reply:
The track exists, the GH branch does not. The charm will be published soon.